### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/Two endpoints in one process/SyncUsersEndpoints/RabbitMessageHandler.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public RabbitMessageHandler(AccountsContext context)
         {
             _context = context;
         }
-        public async Task Handle(NewUser message, IMessageHandlerContext context)
+        public Task Handle(NewUser message, IMessageHandlerContext context)
```
Comment (YogirajA) [inline src/Two endpoints in one process/SyncUsersEndpoints/RabbitMessageHandler.cs]: Good call!

File: src/Two endpoints in one process/Users/Users/Startup.cs [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
+﻿using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 
 namespace Users
 {
-    using Controllers;
```
Comment (YogirajA) [inline src/Two endpoints in one process/Users/Users/Startup.cs]: In general, I like to keep the usings inside the namespace. I will update that later.

File: src/Two endpoints in two processes/SyncUserSubscriber/SyncUserSubscriber.csproj [hunk 3]
```diff
@@ -1,80 +1,15 @@
...
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Messages\Messages.csproj">
-      <Project>{2f770b29-36cd-4885-a98e-5fa8f35dc4fd}</Project>
-      <Name>Messages</Name>
-    </ProjectReference>
-  </ItemGroup>
   <ItemGroup>
-    <Folder Include="Properties\" />
+    <PackageReference Include="NServiceBus" Version="6.*" />
```
Comment (YogirajA) [inline src/Two endpoints in two processes/SyncUserSubscriber/SyncUserSubscriber.csproj]: This is neat. How did you organize this? Is there a new nuget command or some args for Install-Package command? I used install-package and it didn't create this.

File: src/Two endpoints in one process/SyncUsersEndpoints/Program.cs [hunk 4]
```diff
@@ -7,8 +7,7 @@ class Program
     {
         static async Task Main()
         {
-            await RabbitMqEndpoint.StartInstance().ConfigureAwait(false);
-            await SqlServerEndpoint.StartInstance().ConfigureAwait(false);
+            await Task.WhenAll(RabbitMqEndpoint.StartInstance(), SqlServerEndpoint.StartInstance()).ConfigureAwait(false);
```
Comment (YogirajA) [inline src/Two endpoints in one process/SyncUsersEndpoints/Program.cs]: Are you thinking from TPL perspective or from NServiceBus internals?

File: src/Two endpoints in one process/Messages/Messages.csproj [hunk 5]
```diff
@@ -44,10 +44,12 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="NewUser.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
```
Comment (YogirajA) [inline src/Two endpoints in one process/Messages/Messages.csproj]: I'd like to know more on this.

**Comment by SimonCropp** [comment]
> What do you think of this overall? Is this on the right path to achieve autonomy between users app and the accounts app?

TBH this would probable be a better question to ask on the google group

**Pull request merged**